src/main/java/org/bigredbands/mb/controllers/PDFGenerator.java

```diff
@@ -123,9 +123,18 @@ public void createPDF(DrillInfo drillInfo, File file) throws IOException {
                     pageHeight - bufferTop);
             // we have a map of measure number to count per measure
             // currently assumes 4 counts per measure
-            // TODO: Make measures adjust with changes in count per measure
             begMeasure = endMeasure;
-            endMeasure = begMeasure + move.getCounts() / 4;
+            int totalCounts = 0;
+            int currentMeasure = begMeasure;
+            while (move.getCounts() > totalCounts){
```

Comment:
The `while` loop is probably the most readable way to implement this, albeit not the most efficient (it's `O(n)`, where `n` is the number of measures in the drill). That said, efficiency isn't a major concern here; at most, we'll have on the order of hundreds of measures in a single drill file - not enough to cause a noticeable slowdown to a user when exporting. 

Out of curiosity, I considered what a more "efficient" algorithm might look like (e.g. something `O(m log m)`, where `m` is the size of the counts hashmap), but it got _extremely_ convoluted. Any performance benefit isn't worth difficult-to-read code in this case, so I'm happy with the while-loop approach you have here.

My one request, though, is that you make sure to handle the case where a move ends mid-measure (where the end measure of one move _should_ be the start measure of the next). That shouldn't happen in 99% of cases, but it's safer to handle it in case someone - for some reason - ever needs it.